Enigmatry.Entry.AspNetCore.SampleApp/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using FluentValidation;
+using FluentValidation.Results;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
```

Comment:
I don't think we want all of this warnings ignored like this.
We prefer attribute usage with justification.

Reply:
Not sure who is "we". I agree that each suppress should have justification, but in this case the problem is that from 4 violations that had to be suppressed, only one violation makes sense. 

We have general problem with .editorconfig rules being too strict and code formatting is not in sync with these rules. I suggest focusing time and energy on resolving that, and then these suppress attributes are not going to be needed.

Reply:
Ok, I'll be more precise :) @avanwaas spotted that we're not consistent in this regard and suggested for teams to prefer attributes over pragmas (whenever possible). I agreed. I thought we're on the same level for this globally, but I guess just I assumed too much.

I think we agreed on this topic several times already but nobody's taking a lead to make the list of problematic rules and schedule a meeting?